src/index.ts

```diff
@@ -237,7 +242,7 @@ export const dotBackpackToWallet = async (
   }
 
   const matchingUser = users.find(
-    (user) => user.username === dotBackpackUserName
+    (user: any) => user.username === dotBackpackUserName
```

Comment:
Better to fix `users` (`null | Array<BackpackUser>`) - I can fix it if you want though.

Reply:
I added an interface for BackpackUser but it's empty rn

 I don't know what it looks like, I don't know what to use for the jwt